packaging/src/docker/Dockerfile

```diff
@@ -0,0 +1,54 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM openjdk:8-jre
+
+ARG HADOOP_VERSION
+ARG HIVE_VERSION
+ARG TEZ_VERSION
+
+# Install dependencies
+RUN set -ex; \
+  apt-get update; \
+  apt-get -y install curl; \
+  apt-get -y install procps; \
+  rm -rf /var/lib/apt/lists/*
+
+COPY hadoop-$HADOOP_VERSION.tar.gz /opt
+COPY apache-hive-$HIVE_VERSION-bin.tar.gz /opt
+COPY apache-tez-$TEZ_VERSION-bin.tar.gz /opt
+
+RUN tar -xzvf /opt/hadoop-$HADOOP_VERSION.tar.gz -C /opt/ && \
```

Comment:
If we just copy and decompress these files, I think the docker image will be too large.
My suggestion is whether we can do a multi-phase build and then copy the final files?

Reply:
I think we can refer to mr3-hive project:
https://github.com/mr3project/mr3-run-k8s/blob/master/kubernetes/hive/Dockerfile

Reply:
How big is your final docker image? Maybe it's better to describe it like this

Reply:
It's about 3gb.

Reply:
you don't need the archives in the image, you can just copy the final files

Reply:
This image is too big, a good java image should be limited to 500M-1G, too big an image will affect the service starting on different nodes

Reply:
The archives are removed from the image, now the image size is about 1.4G